spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-cloudfoundry/src/main/java/org/springframework/cloud/data/module/deployer/cloudfoundry/CloudFoundryClient.java

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Just curious...does it make sense to break this out into another project like the Receptor client is?  I'd imagine that this would be useful to more than just spring-cloud-data.